nix/ihaskell-notebook.nix

```diff
@@ -0,0 +1,38 @@
+{ compiler ? "ghc864"
+, commonLib ? import ../nix/iohk-common.nix
+, pkgs ? commonLib.pkgs
+, rtsopts ? "-M3g -N2"
```

Comment:
So the heap could grow too much? `-N2` should be enough, if one needs `-N` can be passed in. `-maxN2` would be even better: https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/using-concurrent.html#rts-flag--maxN%20%E2%9F%A8x%E2%9F%A9

Reply:
This was all copy&pasted from other existing projects. If have not  investigated the parameters.
OH !! and `rtsopts` actually not even used anyway ( I just noticed when reading your comment) 
Looks like the relevant lines of code form the other project were lost.
I need to look at that again